vector/src/main/res/layout/fragment_verification_conclusion.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:padding="16dp">
+
+
+    <TextView
+        android:id="@+id/verificationConclusionTitle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="8dp"
+        android:text="@string/sas_verified"
+        tools:text="@string/sas_verified"
+        android:textColor="?riotx_text_primary"
+        android:textSize="20sp"
+        android:textStyle="bold"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/verifyConclusionDescription"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="8dp"
+        tools:text="@string/sas_verified_successful_description"
+        android:text="@string/sas_verified_successful_description"
+        android:textColor="?riotx_text_secondary"
+        app:layout_constraintTop_toBottomOf="@id/verificationConclusionTitle"/>
+
+    <ImageView
+        android:id="@+id/verifyConclusionImageView"
+        android:layout_width="180dp"
+        android:layout_height="180dp"
+        android:layout_gravity="center_horizontal"
+        android:layout_marginTop="16dp"
+        tools:background="@drawable/ic_shield_trusted"
```

Comment:
can be removed, also, attribute are not sorted well, please format all new layout files with attributes sorting on.

Reply:
Done